src/types/types.ts

```diff
@@ -0,0 +1,14 @@
+export type User = {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+};
+
+export type Todo = {
```

Comment:
```suggestion
export interface Todo = {
```
use interface prefer